Material.Ripple/RippleEffect.cs

```diff
@@ -110,6 +120,25 @@ private void RemoveLastRipple() {
             _last = null;
         }
 
+        public void RaiseRipple(double nX = 0.5, double nY = 0.5) {
```

Comment:
I suppose we should make this method to accept coordinates, not the 0-1. And probably other method with explicit naming about percents.

Reply:
Yeah, I think this is great name

Reply:
`RaiseRippleAbsoluteCoord ` added